actionpack/lib/action_controller/metal/implicit_render.rb

```diff
@@ -2,8 +2,14 @@ module ActionController
   module ImplicitRender
     def send_action(method, *args)
       ret = super
-      default_render unless performed?
-      ret
+      return ret if performed?
```

Comment:
Oh of course that makes sense now.

On Sunday, April 5, 2015, George Claghorn <notifications@github.com> wrote:

> In actionpack/lib/action_controller/metal/implicit_render.rb
> <https://github.com/rails/rails/pull/19377#discussion_r27779553>:
>
> > @@ -2,8 +2,14 @@ module ActionController
> >    module ImplicitRender
> >      def send_action(method, *args)
> >        ret = super
> > -      default_render unless performed?
> > -      ret
> > +      return ret if performed?
>
> @sb8244 <https://github.com/sb8244>: Yes, I deleted my suggestion because
> super can cause the action to be performed. It needs to be called before
> the performed? check.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/rails/pull/19377/files#r27779553>.
>


-- 


  Steve Bussey •SalesLoft

•Software Engineer

e steve.bussey@salesloft.com w salesloft.com
<http://salesloft.com/?v=1&utm_expid=76406993-1.E5ncJhwyTAW9uGaR1WdBTA.1?utm_source=Email-Signature-Rescue&utm_medium=Email-Signature&utm_campaign=Email-Signature-Rescue>

a 3423 Piedmont Rd NE, Atlanta, GA 30305

[image: Facebook] <https://www.facebook.com/SalesLoft> [image: Twitter]
<https://twitter.com/SalesLoft> [image: Linkedin]
<https://www.linkedin.com/company/salesloft> [image: wordpress]
<http://blog.salesloft.com/blog/> [image:
http://salesloft.com/prospector/#video]
<http://salesloft.com/prospector/#video>  [image: Software Engineer Logo]
<http://salesloft.com/?v=1&utm_expid=76406993-1.E5ncJhwyTAW9uGaR1WdBTA.1?utm_source=Email-Signature-Rescue&utm_medium=Email-Signature&utm_campaign=Email-Signature-Rescue>